modules/cert-manager-acme-dns01-ambient-gcp.adoc

```diff
@@ -4,14 +4,14 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="cert-manager-acme-dns01-ambient-gcp_{context}"]
-= Configuring an ACME issuer by using ambient credentials on GCP
+= Configuring an ACME issuer by using ambient credentials on {gcp-short}
 
-You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient credentials on GCP. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with Google CloudDNS.
+You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient credentials on {gcp-short}. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with {gcp-full}DNS.
```

Comment:
Based on the use in the file immediately above, this looks like a missing space typo probably
```suggestion
You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient credentials on {gcp-short}. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with {gcp-full} DNS.
```